qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -13,8 +13,9 @@
 import tempfile
 import traceback
 
+from . import util
+from . import coverage
 from authproxy import AuthServiceProxy, JSONRPCException
-from util import *
```

Comment:
I can file a cleanup PR afterwards doing this more broadly; would you guys prefer I (i) revert this diff here, (ii) import the symbols explicitly instead of the `util` namespace, or (iii) leave as-is for now?